specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -1237,12 +1237,27 @@
           "description": "Location of the backend (IP address or FQDN)",
           "type": "string"
         },
-        "privateLinkId" : {
-          "description": "If this backend is private, provide the private link resource Id. Populating this optional field indicates that this backend is 'Private'",
+        "privateLinkResourceId" : {
+          "description": "If this backend is private, provide the Private Link resource Id. Populating this optional field indicates that this backend is 'Private'",
           "type": "string"
         },
-        "location": {
-          "description": "Region of the backend if it is in Azure. Mandatory field if backend is 'Private'",
+        "privateEndpointStatus" : {
+          "description": "The Approval status for the connection to the Private Link",
+          "type": "string",
+          "enum": [
+            "Pending",
+            "Approved",
+            "Rejected",
+            "Disconnected",
+            "Timeout"
+          ],
+          "x-ms-enum": {
+            "name": "PrivateEndpointStatus",
+            "modelAsString": false
+          }
+        },
```

Comment:
this needs to be read only. 
to be able to model this so that the field doesn't exist in update, but show up in PUTs, may need to break up "Backend" into two, one that has the read only field, one that doesn't